docs/es/week03/03-1.md

```diff
@@ -330,26 +329,25 @@ The right side shows the ventral pathway, which tells you what you're looking at
 
 ## [Inspiración de la biología](https://www.youtube.com/watch?v=FW5gFiJb-ig&t=2254s)
 
-¿Por qué el aprendizaje profundo debe estar arraigado en la idea de que nuestro mundo es comprensible y tiene una naturaleza composicional? Estudios conducidos por Simon Thorpe ayudaron a motivar esta pregunta. Él demostró que la manera en que reconocemos objetos cotidianos es muy rápida. Sus experimentos consistieron en mostrar rápidamente un conjunto de imágenes cada 100ms, y enseguida preguntó a sus usuarios que identificaran estas imagenes. Los usuarios fueron capaces de identificar estas sin problema. En el siguiente diagrama se ilustran distintas partes del cerebro indicando el tiempo que toma para que las neuronas propaguen información de un área a otra:
+¿Por qué el aprendizaje profundo debe estar arraigado en la idea de que nuestro mundo es comprensible y tiene una naturaleza composicional? Estudios conducidos por Simon Thorpe ayudaron a motivar esta pregunta. Él demostró que la manera en que reconocemos objetos cotidianos es muy rápida. Sus experimentos consistieron en mostrar rápidamente un conjunto de imágenes cada 100 ms y enseguida preguntar a sus usuarios que identificaran estas imagenes, quienes fueron capaces de hacerlo. Esto demostró que solo le toma 100 ms a los humanos detectar objetos. Adicionalmente, considera el siguiente diagrama, que ilustra las distintas partes del cerebro indicando el tiempo que toma para que las neuronas propaguen información de un área a otra:
 
 <center><img src="{{site.baseurl}}/images/week03/03-1/Simon_Thorpe.png" alt="Simon_Thorpe" style="zoom:55%;" /></center>
 
-<div align="center">Fig. 11. Modelo de flujo de información visual al cerebro de Simon Thorpe <div>
-
-Las señales pasen de la retina al LGN (este ayuda con la mejora del contraste, el  manejo de compuertas, etc.), luego proceden al córtex visual primario V1, V2, V4,  seguido del córtex inferotemporal (PIT), el cual, es la parte del cerebro dónde las categorías están definidas. Observaciones de cirugías cerebrales han mostrado que, si a un humano le es enseñado un filme, las neuronas en el PIT se activaran únicamente cuando detecten ciertas imágenes -- como Jennifer Aniston o la abuelita de alguna persona -- y de ninguna otra forma. Las respuestas neuronales son invariantes a cualidades como posición, tamaño, iluminación, la orientación de la abuelita, o cómo está vestida esta.
+<div align="center">Fig. 11. Modelo de Simon Thorpe de flujo de información visual en el cerebro</div>

```

Comment:
suggestion: "...Modelo de Simon Thorpe **de** flujo..." to "...Modelo de Simon Thorpe **del** flujo"

Reply:
Agreed